Sources/StreamVideoSwiftUI/AnyViewFactory.swift

```diff
@@ -0,0 +1,195 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import StreamWebRTC
+import SwiftUI
+
+public final class AnyViewFactory: ViewFactory {
```

Comment:
That is needed as i can't pass around `any ViewFactory` objects and PiP cannot generic on the ViewFactory type.

Reply:
this is a bit ugly - can't we just pass the `DefaultViewFactory` instead?

Reply:
What is the thing that makes it ugly? if we use the parameter type to DefaultViewFactory noone will be able to customise it (as they will need subclasses of DefaultViewFactory only). I think that the Generic nature of our Views are limiting us right now. Erased types are very useful. However, i understand any concerns and i have a different approach in mind where we can create a small factory for PictureInPicture that can be used to provide configurable UI. That being said introducing more factories doesn't seem to be the solution.

Reply:
For now I replaced it with `PictureInPictureViewFactory` which is a much slimmer version of it.